flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionFIFOITCase.java

```diff
@@ -85,54 +88,75 @@ public void checkForProhibitedLogContents() {
 	public void testDetachedMode() throws InterruptedException, IOException {
 		LOG.info("Starting testDetachedMode()");
 		addTestAppender(FlinkYarnSessionCli.class, Level.INFO);
-		Runner runner =
-			startWithArgs(new String[]{"-j", flinkUberjar.getAbsolutePath(),
-						"-t", flinkLibFolder.getAbsolutePath(),
-						"-n", "1",
-						"-jm", "768",
-						"-tm", "1024",
-						"--name", "MyCustomName", // test setting a custom name
-						"--detached"},
-				"Flink JobManager is now running on", RunTypes.YARN_SESSION);
-
+		File exampleJarLocation = getTestJarPath("StreamingWordCount.jar");
```

Comment:
I think we shouldn't change the example we run as part of this PR. It's somewhat unrelated.

Reply:
We need to add the jar for the new mode, otherwise no TaskExecutor will be brought up.